File: 10 - Python/Exercícios/04 - Vetores/Exercício 08.py
#Exercício - Média Pares

soma = 0
pares = 0

N = int(input("Quantos elementos vai ter o vetor? "))

vet = [0] * N

for i in range(N):

    vet[i] = float(input("Digite um número: "))

    if vet[i] % 2 == 0:
        soma = soma + vet[i]
        pares = pares + 1
    
if pares == 0:
    print("Nenhum número par")
else:
    media = soma / pares
    print("Média dos pares =", round(media,1))

File: 06 - C/Exercícios/04 - Vetores/Exercício 08.c
//Exercício - Média Pares

#include <stdio.h>

int main(){

    int i, N, pares;
    double soma, media;

    soma = 0;
    pares = 0;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d", &N);

    int vet[N];

    for(i = 0; i < N; i++){

        printf("Digite um numero: ");
        scanf("%d", &vet[i]);

        if(vet[i] % 2 == 0){
            soma = soma + vet[i];
            pares++;
        }
    }

    if(pares == 0){
        printf("Nenhum numero par");
    }
    else{
        media = (double) soma / pares;
        printf("Media dos pares = %.1lf", media);
    }
    
    return 0;

}